pkg/notify/notify.go

```diff
@@ -135,11 +97,55 @@ func (n *Notifier) Close() {
 	n.mu.Lock()
 	defer n.mu.Unlock()
 	for _, receiver := range n.receivers {
-		if receiver.rec.ticker != nil {
-			receiver.rec.ticker.Stop()
-		}
-		close(receiver.rec.closeCh)
+		receiver.rec.close()
 	}
 	n.receivers = nil
 	n.closed = true
 }
+
+// Receiver is a receiver of notifier, including the receiver channel and stop receiver function.
```

Comment:
Why changing the location of following code, it will generate too many changes in git diff.

Reply:
To distinguish `Receiver` from`Notifier` code block.